client/src/main/scala/org/http4s/client/middleware/Retry.scala

```diff
@@ -54,7 +50,7 @@ object Retry {
         duration: FiniteDuration,
         retryHeader: Option[`Retry-After`])(
         implicit F: Effect[F],
-        executionContext: ExecutionContext): F[DisposableResponse[F]] = {
+        T: Timer[F]): F[DisposableResponse[F]] = {
```

Comment:
Don't these implicits just shadow the ones in apply?

Reply:
Yes, I didn't pay enough attention to the indentation in this file. Thought they were separate things.